cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -31,17 +31,39 @@ <h1>Find a housing counselor</h1>
                             <div class="o-featured-content-module_text">
                                 <form class="o-form" action="#">
                                     <div class="m-form-field-with-button">
+
                                         <div class="m-form-field">
                                             <label class="a-label a-label__heading" for="hud_hca_api_query">
                                                 Search by ZIP code:
                                             </label>
+
+                                            {% if zipcode and not zipcode_valid %}
+                                            <div class="u-mb15">
+                                                <div class="m-notification
+                                                            m-notification__default
+                                                            m-notification__error
+                                                            m-notification__visible">
+                                                    {{ svg_icon( 'warning-round' ) }}
+                                                    <div class="m-notification_content" role="alert">
+                                                        <div class="h4 m-notification_message">
+                                                            You have entered an invalid ZIP code.
```

Comment:
This bit of error text in the current version of the tool reads "Sorry, you have entered an invalid zip code." I think capitalizing ZIP in the new version makes sense (that seems like an obvious error in the current version), but should we leave "Sorry" in there?

Reply:
Yeah I checked the `zip` with Sarah and it should be capitalized. I removed the "Sorry," and the "Please" because it makes the error notification very wordy. See this versus this:

<img width="1152" alt="screen shot 2018-11-20 at 3 30 35 pm" src="https://user-images.githubusercontent.com/704760/48801034-5c930f00-ecd9-11e8-821f-b21455a130a9.png">

<img width="1148" alt="screen shot 2018-11-20 at 3 30 46 pm" src="https://user-images.githubusercontent.com/704760/48801039-5f8dff80-ecd9-11e8-91ec-70e682fa69f7.png">

Also, what are we apologizing for? The user entered the zip wrong! If anything, the "please" probably makes more sense to retain, but I'd vote we aim for brevity in notifications.

Reply:
I'm not sure of the history of this content and thus am loathe to change it too much. So, for now, let's go with correcting "zip" to "ZIP" but leaving everything else the same: "Sorry, you have entered an invalid ZIP code."

Reply:
Fixed!